src/player/PlayerComponent.cpp

```diff
@@ -164,6 +163,11 @@ bool PlayerComponent::componentInitialize()
   mpv::qt::set_property(m_mpv, "hr-seek", "no");
   // Force vo_rpi to fullscreen.
   mpv::qt::set_property(m_mpv, "fullscreen", true);
+  // wait for cache to be filled before starting playback
+  mpv::qt::set_property(m_mpv, "cache-pause-wait", "yes");
+  // define maximum cache time in seconds
+  mpv::qt::set_property(m_mpv, "cache-pause-secs", 20);
```

Comment:
This isn't a real property, afaik has never been. Not sure what was intended here; if it was `cache-secs`, that defaults to something much higher, which is generally desirable.

Reply:
I think @LongChair may have intended to line up `cache-secs` and `cache-pause-wait` due to the recommendation of the doc above. I see we don't define `cache-secs` anywhere.

I'm planning to remove `cache-pause-secs` and see how our testers fair with remote playback.